packages/components/src/Tabs/Tabs.css

```diff
@@ -1,10 +1,11 @@
 :root {
-  --tab--height: calc(var(--space-larger) + var(--space-small));
-  --tab--inset: var(--space-base);
   --public-tab--inset: var(--tab--inset);
```

Comment:
I'm not too familiar with css variables when it comes to using them before they are defined

is this going to have a value via some sort of hoisting? or I wonder if it's going to say "I don't know who that is" since we define `--tab--inset` below this line. gonna take a peek.

Reply:
ah, yeah it does **not** do any kind of hoisting and so these end up being undefined and we lose the padding right/left. it's tough to see but if you inspect the DOM the highlighting will show ya

![image](https://github.com/GetJobber/atlantis/assets/11843215/d7809672-c658-4d8a-81e9-08ca4e3bed13)

![image](https://github.com/GetJobber/atlantis/assets/11843215/7a0c10f8-6d25-4f81-ac7a-819e46c4efd1)

VS the old way

![image](https://github.com/GetJobber/atlantis/assets/11843215/d86d9585-7dd9-42e6-b3c3-66f6d5a5886c)

Reply:
https://github.com/GetJobber/atlantis/pull/1946/commits/043e747ae904ffd6064b987a261bccd68eb16d56

Reply:
this is an interesting case. the quickest fix is to move the `:root` definition down below `.tabs` though that is kind of gross and breaks the pattern set by every other `:root` definition's placement in the css file

Reply:
nice. ok I think that's a good fix.

I don't quite understand why we want to allow the vertical padding and horizontal padding to diverge but it's already enabling that today so 🤷

Reply:
It's because that public inset is allowing common layout / container components to manipulate whether the tabs's children sit flush or indented to the left/right, which is decoupled from whether the content itself should be flush.

Arguably the tab component should have less of an opinion about that altogether, but I think this is was an early "convenience" solve that's pretty embedded in product usage by this point.